generators/ci-cd/templates/_.gitlab-ci.yml

```diff
@@ -1,21 +1,19 @@
-<%_ if (integrations.includes('docker')) { _%>
```

Comment:
@pascalgrimaud What do you think about separate questions about using Docker? I need gitlab-ci with Docker and Jenkinsfile without Docker.